CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # did-io ChangeLog
 
+## 0.8.0
+(Migrating from v0.7.0 will require renaming the relevant properties in existing
+DID Docs.)
+
+## Changed
+
+- (BREAKING) Updated terminology to match latest VeresOne DID method specs:
```

Comment:
Minor nit, we've been using a bold style markup like:
- \*\*BREAKING**: ...

Reply:
Got it, thanks!